### ISSUE TITLE
Step 6 "Declare database classes" - why should all db classes go into one file?

### ISSUE THREAD

**Issue by pluppplupp** [Starcounter/InvoiceDemo#44]
Referring to: http://starcounter.io/tutorials/1-introduction-to-part-1/1-6-create-the-database-classes/

The tutorial says: "The class definitions are very expressive, that is why we put both definitions in a single file."

Why should all db classes go into one file? Is this a StarCounter best practice? If there will be a lot of db classes this file will get huge? I think it is a .NET guideline to have one class one file, even though it's sometimes ridiculous..

Some discussion about the subject: http://stackoverflow.com/questions/2434990/one-class-per-file-rule-in-net

**Comment by warpech** [comment]
Good point. I don't think we should change the established good practices.

I have changed the text to say: `For the simplicity of the tutorial, we will put both classes in the same file (`Database.cs`), though in your app you might probably want to store them in separate files.`

Maybe I should actually split this into two files? Anyone has strong opinion about this? @dan31, @Starcounter-Jack, @miyconst?

**Comment by dan31** [comment]
In Starcounter samples we prefer condensed code style. That's why we put definitions closer to each other. Otherwise we don't insist, whatever practice people take it is the best one for them if they can ship great software. By our example we show that we prefer less lines of code, and fit more code into a single screen (not sacrificing understanding of course).

**Comment by miyconst** [comment]
I would keep it in one file for the sake of simplicity, and I like your text.
